docs/getting-started/prerequisite.md

```diff
@@ -168,72 +168,74 @@ correct operation of zoned block devices and to troubleshoot problems.
 
 ### lsblk
 
-The lsblk command in Linux lists block devices, including zoned block devices.
-This utility is generally packaged as part of the *util-linux* package which is
-installed by default on most Linux distributions.
+The `lsblk` command in Linux lists block devices. This includes zoned block
+devices.  This utility is usually included in the *util-linux*
+package, which is installed by default on most Linux distributions.
 
 *lsblk* usage examples are provided [here](../linux/utilities/#lsblk).
 
 ### blkzone
 
-Similarly to the `lsblk` utility, the `blkzone` utility is another program
-generally packaged as part of the *util-linux* package. This utility allows
-listing (reporting) the zones of a zoned block device and to reset the write
-pointer position of a range of zones of the device.
+The `blkzone` utility lists (reports) the zones of a zoned block device and
+makes it possible to reset the write pointer position of a range of zones in
+the device. This utility is usually included in the *util-linux* package, which
+is installed by default on most Linux distributions.
 
 *blkzone* usage examples are provided [here](../projects/util-linux.md#blkzone).
 
 ### lsscsi
 
 The <a href="http://sg.danny.cz/scsi/lsscsi.html" target="_blank">*lsscsi*</a>
 command lists information about the SCSI devices connected to a Linux system.
-*lsscsi* is generally available as a package with most Linux distributions.
-Refer to your distribution documentation to find out the name of the package
-providing the *lsscsi* utility.
+*lsscsi* is generally available as a package in most Linux distributions.
+Refer to your distribution documentation to find the name of the package
+that provides the *lsscsi* utility.
 
-The [linux utilities](../projects/sg3utils.md#lsscsi) page provides more information
-and usage examples.
+The [linux utilities](../projects/sg3utils.md#lsscsi) page provides more
+information on `lssci` as well as usage examples.
 
 ### sg3_utils
 
-The <a href="http://sg.danny.cz/sg/sg3_utils.html" target="_blank">*sg3_utils*</a>
-package is a collection of command line tools that send SCSI commands to a SCSI
-device.
+The <a href="http://sg.danny.cz/sg/sg3_utils.html"
+target="_blank">*sg3_utils*</a> package is a collection of command line tools
+that send SCSI commands to a SCSI device.
 
-Since in Linux all disks are exposed as SCSI disks, including all ATA drives,
-these utilities can be used to manage both SAS ZBC disks and SATA ZAC disks.
-For SATA disks connected to SATA ports (e.g. An AHCI adapter), the kernel SCSI
-subsystem translates SCSI commands to ATA commands.
+In Linux, all disks are exposed as SCSI disks. This includes ATA drives.
+`sg3_utils` can be used to manage SAS ZBC disks as well as SATA ZAC disks. When
+dealing with SATA disks connected to SATA ports (for example, an AHCI adapter),
+the kernel SCSI subsystem translates SCSI commands into ATA commands.
```

Comment:
Oops, there is a mistake here. The translation is actually done by the ATA subsystem, in drivers/ata/libata-scsi.c. So this should read:

the kernel ATA subsystem (*libata*) translates SCSI commands into ATA commands.

Reply:
I've made this change.